vnext/JSI/Shared/ChakraRuntime.cpp

```diff
@@ -560,7 +534,9 @@ facebook::jsi::Value ChakraRuntime::call(
   std::vector<ChakraObjectRef> argRefs = ToChakraObjectRefs(args, count);
 
   std::vector<JsValueRef> argsWithThis = ConstructJsFunctionArguments(thisRef, argRefs);
-  assert(argsWithThis.size() <= (std::numeric_limits<unsigned short>::max)());
+  assert(
+      static_cast<unsigned long long>(argsWithThis.size()) <=
```

Comment:
>unsigned long long [](start = 18, length = 18)

nit: I think "long" and "long long" can confuse folks occasionally. What about uint64_t?

Reply:
Yes uint64_t works here. But then if we can use numeric_cast we can get rid of this altogether.

---
In reply to: [345460001](https://github.com/microsoft/react-native-windows/pull/3650#discussion_r345460001) [](ancestors = 345460001)